wgpu-core/src/device/life.rs

```diff
@@ -304,6 +304,7 @@ impl<B: hal::Backend> LifetimeTracker<B> {
         let done_count = self
             .active
             .iter()
+            // TODO: return device lost from `get_fence_status`
             .position(|a| unsafe { !device.get_fence_status(&a.fence).unwrap() })
```

Comment:
I'm not sure how I could get a `Result` out of the `position` closure :thinking:

Reply:
for this case I think it's fine to even have `unwrap_or(false)`